### DISCUSSION TITLE
Any way to get battery level for Logi device connected via "Bolt" USB receiver?

### DISCUSSION THREAD

**Discussion by The111** [Hammerspoon/hammerspoon#3615] [General]
I have a Logitech mouse and keyboard both connected wirelessly via Logi's USB "Bolt" receiver. The Logi Options+ app is able to read battery levels, so obviously the data is there, but I have no idea if this API is usable by Hammerspoon.

**Comment by skefy** [comment]
I found that the third-party mouse driver tool http://better-mouse.com can obtain battery information.
So it should be achievable.

**Comment by cmsj** [comment]
There's probably some open source code out there somewhere which can read the battery levels. It would be interesting to see, to determine how reasonable it would be for us to add.
